apps/builder/app/builder/features/style-panel/controls/select/select-control.tsx

```diff
@@ -16,7 +16,7 @@ export const SelectControl = ({
   const styleValue = currentStyle[property]?.value;
   const setValue = setProperty(property);
   const options = (items ?? defaultItems).map(({ name }) => name);
-  const value = toValue(styleValue);
+  const value = toValue(styleValue) || "advanced";
```

Comment:
In what case toValue not able to provide something?

Reply:
guaranteedInvalid, should  we treat it more explicitely rather than returning an undefined?

Reply:
wait, you added this result there:

  // https://www.w3.org/TR/css-variables-1/#guaranteed-invalid
  if (value.type === "guaranteedInvalid") {
    return "";
  }

Reply:
Do you expect custom properties to appear in select control?

Reply:
no, that's what "advanced" is for

Reply:
it will say in the select "advanced" and hint will tell them to look into advanced section